x-pack/plugins/security_solution/server/endpoint/routes/resolver/tree/queries/lifecycle.ts

```diff
@@ -51,8 +52,15 @@ export class LifecycleQuery extends BaseResolverQuery {
               },
             },
             {
-              term: { 'event.category': 'process' },
+              bool: {
+                should: [
+                  { term: { 'event.category': 'process' } },
+                  { term: { 'event.category': 'malware' } },
```

Comment:
everything looks good except for this change, why add these? The lifecycle events should be comprised of just process events, and this change will change behavior for endpoint/winlogbeat as well

Reply:
I am glad you look at it @kqualters-elastic, thanks! 
So I believe it can introduce changes to endpoint/winlogbeat, however it doesn't really feel wrong to me at this point. Since there are also checks for entity_id, meaning we would be actually fetching properly related data, right? 

The reason for having this is that actually that is how Sentinel One saves the category, but the events contain process details. These also are proper [ECS categories ](https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-category.html). And in order to properly reflect S1 trees, these were required, even more: we might need to add some more. 

However, I know that you have more experience with this so will trust your call, if you think we should do it differently, let's talk :)

Reply:
The reasoning for not having these in this request is the 'node limit' api option. Consider a hypothetical process that a user selects analyzer view, that is particularly noisy. Writing tons of files, network events, etc. By having these categories here, we might ONLY get the 5000+ file events for this process, and never attempt to fetch any ancestors or descendants. I don't think we can do this this way.

Reply:
Can you share examples of the sentinel one data somewhere? I can help figure out an approach that will not break anything ha.

Reply:
As mentioned offline, I actually removed these changes an it still works as expected... so I have to double check this one more time later this week, lets talk next week :) Thanks Kevin!

Reply:
Ok apparently something didn't load properly while recent testing I still need these changes to support other categories. Let's work it out together if possible to find a proper solution :) Thanks @kqualters-elastic 👍